_pages/en/strategy-product-management.md

```diff
@@ -0,0 +1,475 @@
+
+| Workstream              | OPI   | Contributors                         |
+| ----------------------- | ----- | -------------------------------------|
+| <strong>Investment Management</strong> | CFOB EPMO | IITB BPMO, IITB RM, IITB BRM, IITB IT Strategy |
+| <strong>Financial Management</strong> | IITB RM | IITB BPMO, CFOB FMAS |
+| <strong>People Management</strong> | IITB HR | IITB RM, IITB SAs |
+| <strong>Governance</strong> | IITB IT Strategy + CFOB EPMO | IITB BPMO, OCIO (Gov), IITB EA, IITB IM, IT Security (ATOs), IITB CM (Release Mgt) |
+| <strong>Outcome Measurement</strong> | CFOB IPM (Note 1) | SSPB CDO (?), TMB BAAO (?) |
+| <strong>Portfolio Management</strong> | IITB BPMO | IITB BRM, IITB RM, CFOB FMAS |
+| <strong>Change Management</strong> | IITB CM&C | IITB IT Strategy |
+| <strong>Coordination & Mgt of FY 22/23 initiative</strong> | IITB IT Strategy | IITB Resource Centre (?) |
+
+_Note 1_: Potential link with [ESDC Directive on Benefits Management](https://gpp-ppm.service.gc.ca/sites/pwa/ESDCKnowledgeRepository/All%20Documents/Directive%20on%20Benefits%20Management.pdf)
+
+_Note 2_: We seek one candidate funded with A-Base source and one candidate funded with B-Base source.
+
+## Appendix A - Product Architectural Boundaries
+
+A given Product has the following architectural boundaries, used by Enterprise Architecture to provide strategic advice:
+
+- ***Business Alignment***, to the [Business Capability Model](https://dialogue/grp/EA-AE/SitePages/DBCM.aspx).
+- ***Information Profile***, defined by the following attributes:
+  - Privacy, manifested by a [Privacy Impact Assessment](https://www.priv.gc.ca/en/privacy-topics/privacy-impact-assessments/gd_exp_202003/#toc4) (includes Retention and Disposition rules)
+  - Security, manifested by a Statement of Sensitivity and a Statement of Acceptable Risk
+  - Ethics, manifested by one or more Algorithmic Impact Assessment
+- ***Technology Standards***, defined by the following attribute:
+  - Unacceptable technical stacks ([in containment or retirement](https://dialogue/grp/CO7397744/Lists/Standards%20Catalogue/AllItems.aspx))
+
+## Appendix B - Level of Autonomy
+
+The level of autonomy is inspired by the [CFO's financial delegation authority](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32584) and [Heads of HR delegation](https://www.canada.ca/en/public-service-commission/services/appointment-framework/delegation-overview/appointment-delegation-accountability-instrument.html).
+
+The Senior ADM of the IT Branch is both the CIO and the Departmental Official Responsible for Cyber Security (DOCS).
+This person is accountable to [509 policy requirements](https://sara-sabr.github.io/ITStrategy/2021/06/02/stats-digial-policy-instruments.html).
+It is our hypothesis that, in order to have sufficient assurance that policy requirements are met, CIOs and DOCS require IM/IT changes to pass through their central corporate release process, including architecture and security assessments.
```

Comment:
Is it an hypothesis? I thought this was already the case.

Reply:
Yeah, let's just be straight to it ;) removing "hypothesis" part